lib/bcu/options.rb

```diff
@@ -45,6 +48,18 @@ def self.parse!(args)
       opts.on("--no-quarantine", "Add --no-quarantine option to install command, see brew cask documentation for additional information") do
         options.install_options += " --no-quarantine"
       end
+
+      opts.on("--pinned", "List pinned apps") do
+        options.list_pins = true
+      end
+
+      opts.on("--pin APP", "App to pin") do |app|
```

Comment:
Everywhere we / Homebrew uses `CASK`, I don't see a reason why we should introduce new naming? I saw that in one of the versions you had used that, any particular reason why you decided to use different naming?

Reply:
I saw that the code/documentation used `app` in some places, so I change them to `app` to be consistent. But I changed it to `cask` now.